android-sdk/src/main/java/com/optimizely/ab/android/sdk/OptimizelyClient.java

```diff
@@ -794,12 +794,7 @@ public OptimizelyConfig getOptimizelyConfig() {
     @Nullable
     public OptimizelyUserContext createUserContext(@NonNull String userId,
                                                    @NonNull Map<String, Object> attributes) {
-        if (isValid()) {
-            return optimizely.createUserContext(userId, attributes);
-        } else {
-            logger.warn("Optimizely is not initialized, could not create a user context");
-            return null;
-        }
+        return optimizely.createUserContext(userId, attributes);
```

Comment:
```suggestion
if (optimizely != null) {
            return optimizely.createUserContext(userId, attributes);
        } else {
            logger.warn("Optimizely is not initialized, could not create a user context");
            return null;
        }
```
It's a good catch - createContext should be allowed even when optimizely is not fully ready. But we still need null checking.